public/locales/ca-CA/translations.json

```diff
@@ -560,5 +560,7 @@
   "search_results_banner": null,
   "enable_amendment_name": null,
   "amendment_status": null,
-  "expected_date": null
+  "expected_date": null,
+  "transaction_type_name_AMMClawback": null,
+  "action_from_and": null
```

Comment:
Is it possible to use a more informative name for this variable? what does this indicate `action_from_and` ? Could it be `translate_amm_clawback_txn` ?

Reply:
It is generic on purpose so others can re-use this for any other purposes (if you look at other translation variables you will see the same pattern)